src/site/includes/homepage-warning.html

```diff
@@ -2,7 +2,7 @@
   <div id="top-of-page-alert" class="usa-alert usa-alert-warning">
       <div class="usa-alert-body">
           <h4 class="usa-alert-heading">Some VA.gov tools and features may not be working as expected</h4>
-          <p class="usa-alert-text">We’re sorry. We’re working to fix some problems with DS Logon right now. Please check back later or call MyVA311 for help: <a href="tel:18446982311">844-698-2311</a>. If you have hearing loss,
+          <p class="usa-alert-text">We’re sorry. We’re working to fix some problems with DS Logon right now. Please check back later or call VA311 for help: <a href="tel:18446982311">844-698-2311</a>. If you have hearing loss,
```

Comment:
Pretty sure this template isn't used anymore anyway.

Reply:
Could we experiment with deleting it and see if it breaks anything? 🙂

Reply:
We can get that in https://github.com/department-of-veterans-affairs/va.gov-team/issues/8288. I want to knock out this content change quickly.